### PR TITLE
WFLY-14551 EJB subsystem should not allow expressions for cache-container/bean-cache in passivation-store resource and client-mappings-clustering-name in remote resource

### PR DESCRIPTION
**Pull request by rhusar**
Resolves
https://issues.redhat.com/browse/WFLY-14551
https://issues.redhat.com/browse/WFLY-14552

### CODE REVIEW

**Review by pferraro — CHANGES_REQUESTED** [review]
@rhusar It looks like Ejb3SubsystemUnitTestCase.testExpressionInAttributeValue needs to be updated such that it does not expect an expression.

File: ejb3/src/main/java/org/jboss/as/ejb3/subsystem/ClusterPassivationStoreResourceDefinition.java [hunk 1]
```diff
@@ -57,7 +57,8 @@
     static final SimpleAttributeDefinition CACHE_CONTAINER = new SimpleAttributeDefinitionBuilder(EJB3SubsystemModel.CACHE_CONTAINER, ModelType.STRING, true)
             .setXmlName(EJB3SubsystemXMLAttribute.CACHE_CONTAINER.getLocalName())
             .setDefaultValue(new ModelNode(BeanManagerFactoryServiceConfiguratorConfiguration.DEFAULT_CONTAINER_NAME))
-            .setAllowExpression(true)
+            // Capability references should support expressions
+            .setAllowExpression(false)
```
Comment (chengfang) [inline ejb3/src/main/java/org/jboss/as/ejb3/subsystem/ClusterPassivationStoreResourceDefinition.java:61]: the doc "Capability references should support expressions" seems to contradict with the new change. Did you mean "should not"?

Comment (rhusar) [reply ejb3/src/main/java/org/jboss/as/ejb3/subsystem/ClusterPassivationStoreResourceDefinition.java:61]: @chengfang Thanks, yes, that's clearly a typo. Fixed now!

**Comment by rhusar** [comment]
@pferraro Right – I missed one case, fixed now.

**Review by pferraro — APPROVED** [review]

**Review by chengfang — APPROVED** [review]

**Pull request merged**